src/main/webapp/room.js

```diff
@@ -0,0 +1,19 @@
+let players = [];
```

Comment:
Do you plan on using this list at a later point? I see you add the players to it, but don't actually use in anywhere in this PR

In any case, I think it's okay to keep it here, but in the future, it's better to only add a variable in a PR if you're going to use it somewhere in that PR. If it will be used later, it's better to add it in a later PR.

Reply:
@pruess-google No it was just for a visual in my console log, I can remove it